### PR TITLE
TCVP-1208 created ticket-status component

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

[TCVP-1208](https://justice.gov.bc.ca/jira/browse/TCVP-1208)

Added a new ticket-status component to the staff-portal detail screen.

image

This attempts to follow the wireframe as closely as possible.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

No tests since "yarn test" is broken for this project.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/staff-portal/src/app/components/ticket-status/ticket-status.component.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="ticket-status BC-Gov-15px-black-text">
+    <div>Request submitted date: <span>{{dispute.citizenSubmittedDate | date : 'EEEE, MMMM d, y'}}</span></div>
+    <div>Status: <span>{{dispute.status}}</span></div>
+    <div>Assigned to: <span>{{dispute.jjAssigned}}</span></div>
```
Comment (lfdgov) [inline src/frontend/staff-portal/src/app/components/ticket-status/ticket-status.component.html:4]: On the staff portal dashboard, when jjAssigned is blank, null etc. they want us to display 'Unassigned'.

**Review by pbolduc — APPROVED** [review]
LGTM

File: src/frontend/staff-portal/src/app/components/ticket-status/ticket-status.component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  styleUrls: ['./ticket-status.component.scss']
+})
+export class TicketStatusComponent implements OnInit {
+  @Input() public dispute: any; // FIXME: this should be of type Dispute, not "any".
+  
+  constructor() { }
+
+  ngOnInit(): void {
+    // FIXME, remove these custom mappings and use the Dispute object directly
+    this.dispute.citizenSubmittedDate = this.dispute.DateSubmitted;
```
Comment (lfdgov) [inline src/frontend/staff-portal/src/app/components/ticket-status/ticket-status.component.ts:16]: these two issues are due to the dummy data being passed in from the ticket page.  If you update the dummy data on the dashboard to be a little more production ready then you can remove these custom mappings.

Comment (KDNeufeld) [reply src/frontend/staff-portal/src/app/components/ticket-status/ticket-status.component.ts]: Done. Removed these manual mappings to instead just rely on the actual Dispute object properties

File: src/frontend/staff-portal/src/app/components/ticket-status/ticket-status.component.html (hunk 1 above)
Comment (KDNeufeld) [reply src/frontend/staff-portal/src/app/components/ticket-status/ticket-status.component.html]: Done. Defaulted value to "Unassigned".

**Review by lfdgov — APPROVED** [review]

**Pull request merged**